tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/AllWidgetTests.java

```diff
@@ -33,6 +33,7 @@
 		Test_org_eclipse_swt_widgets_List.class, Test_org_eclipse_swt_widgets_Text.class,
 		Test_org_eclipse_swt_widgets_ScrollBar.class, Test_org_eclipse_swt_widgets_Sash.class,
 		Test_org_eclipse_swt_widgets_Tree.class, Test_org_eclipse_swt_widgets_TabFolder.class,
+		org.eclipse.swt.tests.junit.performance.Test_org_eclipse_swt_widgets_Tree.class,
```

Comment:
It is not appropriate to put performance tests into the main tests package.
Unit tests should run as fast as possible, and performance tests are slow by design.

Reply:
Where should I put these tests to run them on CI?

Reply:
To my knowledge, there is no such place currently.
You can introduce new GitHub job maybe. But frankly I would rather not have it at all. Once problems are fixed, they are unlikely to come back. Yet you are going to burn hours of CPU time over years.

Reply:
Can leave them in for now and disable before merge to master?

Reply:
You can keep whatever you want on your branch.
You can create another PR for testing.
If you want *this* PR to be merged, remove tests.

Reply:
Removed